chapter_working_incrementally.asciidoc

```diff
@@ -73,7 +77,7 @@ we might want to recognise users using usernames and passwords,
 we might want to add a longer notes field as well as short descriptions to our list,
 we might want to store some kind of ordering, and so on.
 But we obey another tenet of the agile gospel:  "YAGNI" (pronounced yag-knee),
-which stands for "You ain't gonna need it!"
+which stands for "You ain't gonna need it!".
```

Comment:
i think the exclamation mark counts as sufficient punctuation to end the sentence.  oreilly also have the strange american convention of putting closing punctuation _inside_ quotes or brackets, even if they don't belong there.  it's a USA thing...

Reply:
```suggestion
which stands for "You ain't gonna need it!"
```